gradio/components.py

```diff
@@ -1886,25 +1905,22 @@ def preprocess(self, x: Dict[str, str] | None) -> Tuple[int, np.array] | str | N
         )
         crop_min, crop_max = x.get("crop_min", 0), x.get("crop_max", 100)
         if is_file:
-            file_obj = processing_utils.create_tmp_copy_of_file(file_name)
+            temp_file_path = self.make_temp_copy_if_needed(file_name)
```

Comment:
The streaming audio demo doesn't work now. The front end can't fetch the audio files. 

![stream_audio_tmp_file](https://user-images.githubusercontent.com/41651716/205978455-6f40e76f-0cd8-423e-a8f4-1c970c104a99.gif)

Reply:
I don't think Audio is playable now

<img width="1316" alt="image" src="https://user-images.githubusercontent.com/41651716/205979926-41fb674a-c719-4a3c-8537-9a6320cab3ac.png">

Reply:
Both of these work for me now

Reply:
It's working on spaces but not for me locally. I'm getting the following for audio debugger

```ValueError: File cannot be fetched: /var/folders/t1/j7cmtcgd0mx43jh9nj_r9mmw0000gn/T/cantina92aff6ffe140da201a4a94cb3c3b9e1ff0b2a25a.wav. All files must contained within the Gradio python app working directory, or be a temp file created by the Gradio python app.
```

I'm wondering if it has anything to do with previous runs of this demo on other branches on my machine

Reply:
Can't reproduce this. Is this with caching enabled?

Reply:
Oh that could be it! Will check when I’m back home. Disregard for now 😄